### PR TITLE
Issue #803 fix ‘xAPI specification’ references sections 8.1.2, 8.2.2 and 8.3

### PR DESCRIPTION
**Pull request by MartinKoob**
Issue # This includes  ‘xAPI specification’ references sections 8.1.2, 8.2.2 and 8.3 as per Aug 18, 2023 meeting.

As well changes were made to definitions of xAPI and use of words 'specifications'  vs Standard

### CODE REVIEW

**Comment by MartinKoob** [comment]
There were additional  instances of  referring to ‘the xAPI specification’ that I found in the document after the Aug 19, 2023 meeting.  I have also changed these instances  to ‘xAPI’.



# 3.0 Definitions 

* __Internationalized Resource Locator (IRL)__: According to the xAPI specification, an IRL is an IRI that, when translated into a URI (according to the IRI to URI rules), is a URL.

* __Learning Records Store (LRS)__: As defined in the xAPI specification.


## 9.1 Statement ID
The AU MUST assign a statement `id` property in UUID format (as defined in the xAPI specification) for all statements it issues.

##9.5.1 Score
The following are properties of the <code>score</code> object per the xAPI specification: 
<ul>

### 10.2.1 contextTemplate

Data Type:	JSON context object as defined in the xAPI Specification.

# 11.0 xAPI Agent Profile Data Model  

In cmi5, Learner Preferences are scoped to the Agent representing the enrolled LMS learner. The agent used in xAPI Agent Profile requests MUST match the actor property generated by the LMS at AU launch time.  Both the LMS and the AU MAY write changes to Learner Preferences in the xAPI Agent Profile.  The LMS/LRS MAY choose to ignore or override Learner Preference changes requested by the AU by returning a "403 Forbidden" response as defined in the xAPI specification.  The AU MUST NOT treat the 403 response as an error condition.

 # 12.0 xAPI Activity Profile Data Model

The AU MAY use the Activity Profile API according to the xAPI specification.

**Comment by MrBillMcDonald** [comment]
Per 8/25 Meeting change xAPI reference link to 

change xAPI reference link to https://github.com/adlnet/xAPI-Spec
Change “xAPI specification” references to just “xAPI” (not “xAPI standard”)

File: cmi5_spec.md [hunk 1]
```diff
@@ -251,10 +251,10 @@ The following referenced documents are indispensable for the application of this
 
 * Internationalized Resource Identifiers (IRIs): IRI Syntax January 2005  
 https://www.ietf.org/rfc/rfc3987.txt
-* "Experience API", version 1.0.x (subject to change just prior to release), ADL  
-https://github.com/adlnet/xAPI-Spec/blob/xAPI-1.0.2/xAPI.md
+* Experience API (xAPI), version 1.0.3  
+https://github.com/adlnet/xAPI-Spec
```
Comment (thomasturrell) [inline cmi5_spec.md:255]: @andyjohnson Is this the correct location? Should we be linking to https://opensource.ieee.org/xapi/xapi-base-standard-documentation?

**Review by thomasturrell — APPROVED** [review]

File: cmi5_spec.md (hunk 1 above)
Comment (thomasturrell) [reply cmi5_spec.md:255]: I now understand that this should reference version 1.0.3 not version 2.0

**Comment by MrBillMcDonald** [comment]
Approved Per Sept 1, Voting meeting

**Pull request merged**